fabric-server-languages-v1/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-server-languages-v1"
+version = getSubprojectVersion(project, "0.1")
+
+moduleDependencies(project, [
+		'fabric-api-base'
```

Comment:
Unnecessary, consider removing this `moduleDependencies` block.

Reply:
Removed